dkg/mod.go

```diff
@@ -26,5 +27,5 @@ type Actor interface {
 	Encrypt(message []byte) (K, C kyber.Point, remainder []byte, err error)
 	Decrypt(K, C kyber.Point) ([]byte, error)
 
-	Reshare() error
+	Reshare(T_new int, T_old int, addrs_new []mino.Address, pubkeys_new []kyber.Point, pubkeys_old []kyber.Point) error
```

Comment:
- is it necessary to specify the old threshold ? We should already know it
- same for the old public keys, we should already know them
- [the convention in Go is to use MixedCaps or mixedCaps rather than underscores](https://go.dev/doc/effective_go#mixed-caps)

The function should pretty much look the same as the `Setup` one:

```go
Reshare(co crypto.CollectiveAuthority, threshold int)
```